marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/LoadEntityModelsCommand.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2012-2019 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.marklogic.hub.deploy.commands;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.marklogic.appdeployer.command.AbstractCommand;
+import com.marklogic.appdeployer.command.CommandContext;
+import com.marklogic.appdeployer.command.SortOrderConstants;
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.datamovement.WriteEvent;
+import com.marklogic.client.document.DocumentWriteSet;
+import com.marklogic.client.document.JSONDocumentManager;
+import com.marklogic.hub.HubConfig;
+import com.marklogic.hub.deploy.util.EntityDeploymentUtil;
+
+@Component
+public class LoadEntityModelsCommand extends AbstractCommand {
```

Comment:
I know this was merged already, but I think it's worth adding a comment to this class to explain why it exists and why it's separate from the commands for loading modules (I know there are good reasons for that, but those probably wouldn't be apparent to a typical reader of this code).